index.html

```diff
@@ -0,0 +1,312 @@
+        </article>
+        <div class="member-count">
+
+        </div>
+        <article class="latest-articles-items">
+          <p class="latest-articles-days">13 DAYS AGO</p>
+          <h2><a class="latest-articles-title" href="#">Releasing The Ethical Design Handbook When We Needed It Most</a></h2>
+          <p class="latest-articles-author">
+            by
+            <a href="#">Ari Stiles</a>
+          </p>
+          <p class="latest-articles-text">Early this month, The Ethical Design Handbook joined the Smashing Library of essential books for designers and developers. Changes around the world have meant changes in the book release, but readers love it so far! Ethical Design principles are …
+          </p>
+        </article>
+        <article class="latest-articles-items">
+          <p class="latest-articles-days">14 DAYS AGO</p>
+          <h2><a class="latest-articles-title" href="#">Django Highlights: Templating Saves Lines (Part 2)</a></h2>
+          <p class="latest-articles-author">
+            by
+            <a href="#">Philip Kiely</a>
+          </p>
+          <p class="latest-articles-text">In this article, Philip explores breaking down a complex webpage into multiple templates, composing those components, and applying tags and filters to refactor a plain HTML page.</p>
+        </article>
+        <article class="latest-articles-items">
+          <p class="latest-articles-days">15 DAYS AGO</p>
+          <h2><a class="latest-articles-title" href="#">Building A Web App With Headless CMS And React</a></h2>
+          <p class="latest-articles-author">
+            by
+            <a href="#">Blessing Krofegha</a>
+          </p>
+          <p class="latest-articles-text">This article introduces the concept of the headless CMS, a backend-only content management system that allows developers to create, store, manage and publish the content over an API.</p>
+        </article>
+        <article class="latest-articles-items">
+          <p class="latest-articles-days">16 DAYS AGO</p>
+          <h2><a class="latest-articles-title" href="#">What Should You Do When A Web Design Trend Becomes Too Popular?</a></h2>
+          <p class="latest-articles-author">
+            by
+            <a href="#">Suzanne Scacca</a>
+          </p>
+          <p class="latest-articles-text">This guide aims to help you figure out which approach makes the most sense when considering popular design trends for your site.</p>
+        </article>
+        <div>
+          <div class="smashing-tv">
+
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="latest-link">
+      <a class="links-a" href="#">SEE ALL LATEST ARTICLES <i class="fas fa-long-arrow-alt-right"></i></a>
+    </div>
+
+    <div class="design-handbook">
```

Comment:
This div and other ones are still missing some visual content. You don't need to literally copy the illustration and make it grey. You can just put the missing text and divs to represent any illustration or picture.